pageserver/src/task_mgr.rs

```diff
@@ -147,7 +147,7 @@ pub(crate) static BACKGROUND_RUNTIME_WORKER_THREADS: Lazy<usize> = Lazy::new(||
     // else, but that has not been needed in a long time.
     std::env::var("TOKIO_WORKER_THREADS")
         .map(|s| s.parse::<usize>().unwrap())
-        .unwrap_or_else(|_e| usize::max(1, num_cpus::get()))
+        .unwrap_or_else(|_e| usize::max(2, num_cpus::get()))
```

Comment:
Does it have any relation to sharding and this PR?

Reply:
Not directly - it came up in the context of testing sharding because I was working with a neon_local setup where I had many pageservers pinned to cores.